code/modules/clothing/suits/armor.dm

```diff
@@ -334,6 +334,21 @@
 	price_tag = 600
 	//Used ablative gear armor values and technomancer helmet/voidsuit values.
 
+//Makeshift chest rig. Uses the merc path to allow it to have 4 pockets without making more clutter.
+/obj/item/clothing/suit/storage/vest/merc/chestrig
```

Comment:
I'm sure you can make it store more than 2 items. Also, why ``merc`` is in a path?

Reply:
Merc vest children initialize with 4 slots rather than 2, I'd have to add another pocket variant just for this which I find a bit odd considering merc rig effectively does it for what I want. But I guess I could do it.

Reply:
Merc children start with 4 slots due to using the new proq that makes it have 4 slots. I don't really see the point of cluttering it more just for this.